woocommerce/woocommerce-android
# feat: add product bulk editing icon

Closes: #7928 


### Description

This PR adds an icon of multiselection to product lists, visibly only on debug builds. The icon is for now hidden when user is in "Search view" as we don't focus on this part right now.

### Testing instructions

Install the app and see if the icon is visible. Check if it's hidden in search view.

### Images/gif

<img width="300" alt="image" src="https://user-images.githubusercontent.com/5845095/203781164-2096ffb7-cf4f-443a-996e-5e9211436ab1.png">
